### PR TITLE
Remove build type flag for in progress applications on profile page

### PR DESCRIPTION
**Pull request by jbalboni**
@rtluu said we can release this now, so this PR removes the build type flag.

### CODE REVIEW

File: src/js/common/schemaform/FormApp.jsx [hunk 1]
```diff
@@ -141,7 +141,7 @@ class FormApp extends React.Component {
       const isPrefill = props.prefillsAvailable.includes(currentForm);
       const saveEnabled = !this.props.formConfig.disableSave;
       if (saveEnabled && (isSaved || isPrefill)) {
-        props.fetchInProgressForm(currentForm, props.formConfig.migrations, isPrefill);
+        props.fetchInProgressForm(currentForm, props.formConfig.migrations, !isSaved && isPrefill);
```
Comment (jbalboni) [inline src/js/common/schemaform/FormApp.jsx]: I found this when testing. The prefill message was being shown for saved apps when you use the `/resume` route. We know if there's a saved application, so we can tell it to not prefill in that case.

Comment (cehsu) [reply src/js/common/schemaform/FormApp.jsx]: Good catch, should we move this logic to line 141 when declaring isPrefill?

File: src/js/user-profile/containers/UserProfileApp.jsx [hunk 2]
```diff
@@ -32,10 +32,10 @@ class UserProfileApp extends React.Component {
         <div className="usa-width-two-thirds medium-8 small-12 columns">
           <h1>Your Account</h1>
           <div>
-            {(__BUILDTYPE__ !== 'production') && <FormList
+            <FormList
```
Comment (cehsu) [inline src/js/user-profile/containers/UserProfileApp.jsx]: Should the tests for UserProfileApp be updated to reflect this? Or are the tests for FormList sufficient and that would that be a case of testing React and not our code?

**Review by cehsu — APPROVED** [review]
LGTM. Had two minor questions.

File: src/js/common/schemaform/FormApp.jsx (hunk 1 above)
Comment (jbalboni) [reply src/js/common/schemaform/FormApp.jsx]: I think the logic is clearer the way it is now, but I did change the prefill variable name to make it clearer what's being checked.

File: src/js/user-profile/containers/UserProfileApp.jsx (hunk 2 above)
Comment (jbalboni) [reply src/js/user-profile/containers/UserProfileApp.jsx]: Added a saved form to the mock data and a basic test.

**Pull request merged**